.gitignore

```diff
@@ -1,4 +1,14 @@
 __pycache__
 installer
 installer.tar
-dist
\ No newline at end of file
+
+# Editor directories and files
+.vscode/*
+!.vscode/extensions.json
+.idea
+.DS_Store
+*.suo
+*.ntvs*
+*.njsproj
+*.sln
+*.sw?
```

Comment:
these were all things that the vite app added to the interior .gitinore moved them to the top level. 

Obviously we can 🔪 if we don't care about it